### DISCUSSION TITLE
Confusion...

### DISCUSSION THREAD

**Discussion by yodaluca23** [SideStore/SideStore#114] [Q&A] [answered]
So if I were to clone this and compile it into an IPA and then sideload it onto my iPhone, using Sidloadly, then can I do everything that AltStore can, Sideload, refresh, activate/deactivate, etc on cellular data without my computer? Essentially making it everything fully sideloadable on device only?

Also, does this have "untrusted sources" option?

**Comment by Spidy123222** [comment, marked as answer]
> So if I were to clone this and compile it into an IPA and then sideload it onto my iPhone, using Sidloadly, then can I do everything that AltStore can, Sideload, refresh, activate/deactivate, etc on cellular data without my computer? Essentially making it everything fully sideloadable on device only?
> 
> 
> 
> Also, does this have "untrusted sources" option?

Yes but it has bugs as it not a full on release rn. You also need a wireguard config for it to work and yes untrusted sources is available. It's based on AltStore 1.5.1b2. You can fully sideload/deactivate/activate on device and activate jit manually atm on the app. 

There is no way to use on cellular atm for sideloading but methods are being tested. you can't use cellular directly as apple restricts lockdown traffic. There is other possible methods that can be used like airplane mode but wireguard atm stops handshake once I touches airplane mode while self hosted vpn like jitterbug can.

**Reply by yur1xpp** [reply]
@Spidy123222 could you explain more on this?

> There is other possible methods that can be used like airplane mode but wireguard atm stops handshake once I touches airplane mode while self hosted vpn like jitterbug can.


Though I didn't use SideStore specifically, but your comment seems related to my setup. I have my own wireguard server, with AltStore-Linux+Provision+netmuxd running on it. When you said _while self hosted vpn like jitterbug can_, what do you means actually? Could my setup be used to take advantage of that?

**Reply by Spidy123222** [reply]
> @Spidy123222 could you explain more on this?
> 
> 
> 
> > There is other possible methods that can be used like airplane mode but wireguard atm stops handshake once I touches airplane mode while self hosted vpn like jitterbug can.
> 
> 
> 
> 
> 
> Though I didn't use SideStore specifically, but your comment seems related to my setup. I have my own wireguard server, with AltStore-Linux+Provision+netmuxd running on it. When you said _while self hosted vpn like jitterbug can_, what do you means actually? Could my setup be used to take advantage of that?

No you would have to use jitterbug with paid entitlements to get that working. Then you just use the jitterbug vpn and put it in airplane mode or turning off cellular to get it to work without Wi-Fi.

**Reply by yodaluca23** [reply]
> > Though I didn't use SideStore specifically, but your comment seems related to my setup. I have my own wireguard server, with AltStore-Linux+Provision+netmuxd running on it. When you said _while self hosted vpn like jitterbug can_, what do you means actually? Could my setup be used to take advantage of that?
> 
> No you would have to use jitterbug with paid entitlements to get that working. Then you just use the jitterbug vpn and put it in airplane mode or turning off cellular to get it to work without Wi-Fi.

I have a paid cert on one of my devices, and have been using Jitterbug for JIT, that would've been good to know that I can do it without Wi-Fi... lol
